doc/user/commands.rst

```diff
@@ -8,14 +8,25 @@ Commands
 
 **hop-client** provides a command line interface for various tasks:
 
-:code:`hop publish`: parse and publish GCN circulars
-
+:code:`hop publish`: parse and publish GCN circulars and notices
+:code:`hop subscribe`: parse and print GCN circulars and notices
 
 :code:`hop publish`
 ~~~~~~~~~~~~~~~~~~~~~~
 
-This command allows a user to parse an RFC 833 formatted GCN circular and
-publish JSON-formatted GCNs via Kafka.
+This command allows a user to parse an `RFC 833 formatted GCN circular <https://gcn.gsfc.nasa.gov/gcn3_circulars.html>`_
```

Comment:
Below it's described as RFC 822. Which one is right? I'm not sure.

Reply:
That's a typo, it should be RFC 822.